pythonie/pythonie/settings/configure.py

```diff
@@ -16,6 +16,11 @@ def configure_redis(redis_url, test=False):
 
         url = urlparse(redis_url)
         log.info('Redis configured with redis_url: %s' % redis_url)
-        return redis.Redis(host=url.hostname, port=url.port, password=url.password)
+        url_details = {
+            'host': url.hostname,
+            'port': url.port,
+            'password': url.password or '',
+        }
+        return redis.Redis(**url_details)
```

Comment:
No need to use a dictionary here. Regular kwargs are fine. Albeit I admit, this does fix the pep8 violation :+1: